ccd-definition/CaseEventToFields/uploadDocuments-SDO_READY_TO_SEND.json

```diff
@@ -0,0 +1,135 @@
+[
```

Comment:
was there not a change to make upload documents available in all states? Is this needed?

Reply:
Good point, did not see that change. Making change now.